### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by mrT23**

### CODE REVIEW

**Comment by mrT23** [comment]
Preparing pr description...

**Comment by mrT23** [comment]
## PR Title

Minor Fixes in PR Agent Tools

## Type of PR

Refactoring

## PR Description

This PR includes minor refactoring changes in the PR agent tools. The main change is the renaming of the 'publish_review' setting to 'publish_output' for better clarity. Additionally, a new setting 'publish_description_as_comment' is introduced in the PR description tool. The PR also includes changes to the verbosity level and the way PR answers are prepared and published.

## PR Main Files Walkthrough

-`pr_agent/tools/pr_code_suggestions.py`: The 'publish_review' setting is renamed to 'publish_output'.
-`pr_agent/tools/pr_description.py`: The 'publish_review' setting is renamed to 'publish_output'. Also, the method '_prepare_pr_answer' is updated to prepare markdown text and the way the PR description is published is changed based on the new 'publish_description_as_comment' setting.
-`pr_agent/tools/pr_information_from_user.py`: The 'publish_review' setting is renamed to 'publish_output'.
-`pr_agent/tools/pr_questions.py`: The 'publish_review' setting is renamed to 'publish_output'.
-`pr_agent/tools/pr_reviewer.py`: The 'publish_review' setting is renamed to 'publish_output'.
-`pr_agent/settings/configuration.toml`: The 'publish_review' setting is renamed to 'publish_output', the verbosity level is changed, and a new setting 'publish_description_as_comment' is introduced.

**Comment by mrT23** [comment]
Preparing review...

**Comment by mrT23** [comment]
## PR Analysis

- 🎯 **Main theme:** The PR mainly involves renaming a configuration variable and adding a new configuration option for publishing PR descriptions. It also includes minor changes to logging and output publishing in various modules.
- 📌 **Type of PR:** Refactoring
- 🧪 **Relevant tests added:** No
- ✨ **Focused PR:** No, because the PR title 'Minor fixes' is not descriptive enough to understand the changes made in the PR. Also, the PR description is not provided which makes it difficult to understand the context and purpose of the changes.
- 🔒 **Security concerns:** No, because the changes in this PR are mostly related to configuration and logging, and do not seem to introduce any potential security issues.
## PR Feedback

- 💡 **General PR suggestions:** The PR includes changes in multiple files and it would be helpful to have a more descriptive title and a detailed PR description explaining the reason for these changes. It's also recommended to add relevant tests for the changes made in the PR to ensure they work as expected.

**Review by hussam789 — APPROVED** [review]

File: pr_agent/settings/configuration.toml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [config]
 model="gpt-4-0613"
 git_provider="github"
-publish_review=true
-verbosity_level=2 # 0,1,2
+publish_output=true
+verbosity_level=0 # 0,1,2
```
[the remark this answers is not in the document]
Comment (hussam789) [reply pr_agent/settings/configuration.toml:5]: @mrT23 Intentional or mistake?

**Pull request merged**